src/chunk_constraint.h

```diff
@@ -11,9 +11,11 @@
 #include "ts_catalog/catalog.h"
 #include "hypertable.h"
 
+#define CC_DIM_PREFIX "_$CC_" /* Correlated Constraint dimension */
 typedef struct ChunkConstraint
 {
 	FormData_chunk_constraint fd;
+	bool correlated;
```

Comment:
Would be good to replace with enum of, e.g., CONSTR_TYPE_CORRELATED, CONSTR_TYPE_DIMENSION, CONSTR_TYPE_REGULAR.

Reply:
@erimatnor Ok done. Added enums